### PR TITLE
Raspberry Pi

### PR DESCRIPTION
**Pull request by ThePigPeppa**

### CODE REVIEW

**Comment by ThePigPeppa** [comment]
@angristan

File: wireguard-install.sh [hunk 1]
```diff
@@ -86,6 +87,14 @@ if [[ "$OS" = 'ubuntu' ]]; then
...
     apt-get install "linux-headers-$(uname -r)"
     apt-get install wireguard iptables
+elif [[ "$OS" = 'raspbian' ]]; then
+    apt-get update
+    echo "deb http://deb.debian.org/debian/ unstable main" >/etc/apt/sources.list.d/unstable.list
+    apt-get install dirmngr -y
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
+    printf 'Package: *\nPin: release a=unstable\nPin-Priority: 90\n' >/etc/apt/preferences.d/limit-unstable
+    apt-get update
+    apt-get install wireguard raspberrypi-kernel-headers -y
```
Comment (angristan) [inline wireguard-install.sh]: missing iptables dependency

**Comment by ThePigPeppa** [comment]
please approve this and ill create a PR in the next day or two with everything, ill take like an hour and rewrite this entire thing.

**Comment by angristan** [comment]
See my changes request :)

**Pull request closed without merging**